samples/winapi_drivelist/main.c

```diff
@@ -9,25 +9,24 @@ int main(void)
 
     // Mount some drives for demonstration purposes
     BOOL ret;
+    DWORD error;
     ret = nxMountDrive('C', "\\Device\\Harddisk0\\Partition2\\");
     if (!ret) {
-        debugPrint("Failed to mount C: drive!\n");
-        Sleep(5000);
-        return 1;
+        // Additional error info can be retrieved with GetLastError()
+        error = GetLastError();
+        debugPrint("Failed to mount C: drive! Error code: %x\n", error);
```

Comment:
The old code simply quit the program (which makes us unable to properly unmount),  but this code now just continues as if the error didn't happen.

I think this is a case where using `goto` is actually beneficial. I'd recommend to add a `goto end;` or something here, that jumps to a label placed just after unmounting the two drives, and a `goto unmount_c;` in the if statement below, which jumps to a label placed after unmounting E and before unmounting C (note that the unmounting order will have to change for that).

It's probably also a good idea to introduce an `unmount_e` label, too, and jump to that in case `GetLogicalDrives()` or `GetLogicalDriveStringsA()` fail.

This similarly applies to the other sample.